### ISSUE TITLE
Missing help on how to quit the xserver

### ISSUE THREAD

**Issue by tomtom** [corbinlc/GNURootDebian#124]
Since the last update, there now is a nice button to quickly jump into a X session but there is no hint as to how users can get out of the X sessions. Selection quit from the twm menu, quits twm but not the X session.

**Comment by Root-voidX** [comment]
I usually close all application on xterm including the bash and just press my home button to go back to android. Then I just manually close the terminal. I had no errors doing so even before this patch. Just a side question, after the update do you have two gnuroot application in your app drawer? One works the other crashes :/

**Comment by tomtom** [comment]
There actually are two icons.

**Comment by Root-voidX** [comment]
Yeah it maybe a problem, since only one works the other crashes ;/

**Comment by Root-voidX** [comment]
Recent update fixed the duplicate icon issue

**Comment by Root-voidX** [comment]
When in X term if you click the options icon besides the keyboard icon there is ano option to disconnect. I can't believe I over looked this!

**Comment by quole** [comment]
After disconnecting you still have to close the associated terminal tab (which is created automatically) to actually kill the X server completely. There ought to be a menu option to "Quit X", which disconnects and also closes associated terminal tab.

**Comment by Root-voidX** [comment]
Well yeah after you disconnect you would have to close the terminal window as well. There is no issue in doing so if you have killed all the pid of apps you ran or close them and saved your data!

**Comment by AdamMatthewLTM** [comment]
Hi all, 

I am attempting to assist the developers in sorting out reported issues on GitHub for GNU Root Debian. @tomtom , could you please let me know if exiting from an X session is still a relevant issue at the moment? Also, if there are current issues that any of you would consider to be high priority, please create a new post on GitHub and I will sort them ASAP.

Current high priority issues we are aware of that do not need duplicates:
- Directory Permissions
- SD Card Issues

Thank you all so much for your time and patience in these matters,
-Adam
